UOP1_Project/Assets/Scripts/Editor/GameSceneEditor.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+[CustomEditor(typeof(GameScene), true)]
+public class GameSceneEditor : Editor
+{
+    #region UI_Warnings
+
+    private string noScenesWarning = "There are no scenes set for this level yet! Add a new scene with the dropdown above";
```

Comment:
Can be a `const` variable since the value does not change

Reply:
Actually, yeah. Gonna update it right now